storage-management/storage-management.md

```diff
@@ -0,0 +1,85 @@
+---
+description: This article explains how to manage and free up your storage on SharePoint sites.
+---
+
+# Storage Management
+
+## Storage Management in Microsoft 365
+
+**Storage management in Microsoft 365 is important, as it directly impacts your monthly expenses**. 
```

Comment:
formatirao bih ovaj dio u bullete gdje u svakom piše nešto bitno
tipa 
There are multiple reasons why you should strategically approach Storage Management in M365:
- Monthly expenses are directly linked...
- When you run out of storage, you must buy...